crates/indexer/src/models/token_models/v2_token_datas.rs

```diff
@@ -71,10 +73,21 @@ impl TokenDataV2 {
         if let Some(inner) = &TokenV2::from_write_resource(write_resource, txn_version)? {
             let token_data_id = standardize_address(&write_resource.address.to_string());
             // Get maximum, supply, and is fungible from fungible asset if this is a fungible token
-            let (maximum, supply, is_fungible_v2) = (None, BigDecimal::zero(), Some(false));
+            let (mut maximum, mut supply, mut decimals, mut is_fungible_v2) =
```

Comment:
required for fungible assets. you know how you can have decimals in coins, e.g. .5 apt. Since move doesn't support decimals we use this to represent how many 10s to divide by.